packages/angular_devkit/build_angular/src/angular-cli-files/utilities/index-file/augment-index-html_spec.ts

```diff
@@ -49,6 +49,24 @@ describe('augment-index-html', () => {
     `);
   });
 
+  it('should emit correct when base-href is present', async () => {
```

Comment:
Maybe `should replace base href when index content is one liner`?

Reply:
This test case is when base tag is present in `index.html`. How about `should replace base href value`?

Reply:
SGTM